registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/validator/property/AdapterIdValidator.java

```diff
@@ -0,0 +1,23 @@
+package de.adorsys.registry.manager.service.validator.property;
+
+import de.adorsys.registry.manager.service.model.AspspBO;
+import de.adorsys.registry.manager.service.model.AspspValidationReportBO;
+
+import java.util.regex.Pattern;
+
+public class AdapterIdValidator implements PropertyValidator {
+    private static final String ADAPTER_ID_REGEX = ".{2,}-adapter";
```

Comment:
Recalling my conversation with @spetrychenko, AdapterId should consist of letters, numbers, and the hyphen only. If it's still true, I'd recommend adjusting the regexp rule to be compliant with that constraint.  Also, should `adapter` be always at the rear?

Reply:
```
AdapterId should consist of letters, numbers, and the hyphen only
```
Good point, thanks, I'll adjust my regex.

```
Also, should adapter be always at the rear?
```
At least we have such a constraint in `xs2a-adapter` app during the new adapter generation. So it seems to me, that we should be compliant with its rules.

Reply:
Done, please, take a look